pages/admin/content/edit.jsx

```diff
@@ -8,16 +9,26 @@ const Edit = ({ content, typeSchema }) => (
     initialValues={content}
     typeSchema={typeSchema}
     onSave={values => {
-      save(content.type, values.slug, values);
+      values.slug = values.slug.trim();
+      debugger;
+      save(content.type, content.slug, { type: content.type, ...values }).then(
+        response => {
+          Router.push('/admin/content/list?contentType=' + content.type);
```

Comment:
keep in mind `'/admin/content/list?contentType=' + content.type` is most likely not the final url path and most likely will change when we start the clean up. we might need a helper in the future that would generate urls by key and params. a rough example of that: `getUrl('contentTypeLanding', contentType)`